### PR TITLE
add demo for the broken stream

### PR DESCRIPTION
**Pull request by gangtao**
add example to show how to use EMIT PERIODIC time_interval REPEAT to monitor broken stream

### CODE REVIEW

File: examples/broken-stream-monitor/script/iot.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    SELECT count(*) AS count FROM device_reader EMIT PERIODIC 2s REPEAT
+)
+SELECT count, lag(count) AS previous_count FROM accumulate_count;
+
+-- alert on broken stream
+WITH accumulate_count AS (
+    SELECT count(*) AS count FROM device_reader EMIT PERIODIC 2s REPEAT
+)
+SELECT count, lag(count) AS previous_count FROM accumulate_count
+WHERE count == previous_count;
```
Comment (jovezhong) [inline examples/broken-stream-monitor/script/iot.sql]: =, not ==

File: examples/broken-stream-monitor/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+
+# Demo for broken stream monitoring using global aggregation
+
+This demo shows how to leverage the new feature of proton SQL to monitor a broken stream, traditionally using watermark, the stream processing will generally stop processing when there is no new event which push the watermark farward.
```
Comment (jovezhong) [inline examples/broken-stream-monitor/README.md]: not common to say `proton SQL`

Comment (jovezhong) [reply examples/broken-stream-monitor/README.md]: no new event which pushes..

File: examples/broken-stream-monitor/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+In this demo, we show how to use the new settings `EMIT PERIODIC time_interval REPEAT` to emit existing result to help monitor stream status.
```
Comment (jovezhong) [inline examples/broken-stream-monitor/README.md]: EMIT is not really a setting

File: examples/broken-stream-monitor/README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+here is the steps
```
Comment (jovezhong) [inline examples/broken-stream-monitor/README.md]: cannot use is and steps. Also here to Here

File: examples/broken-stream-monitor/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+1. create a simulated data source using random stream
```
Comment (jovezhong) [inline examples/broken-stream-monitor/README.md:10]: a stream or streams

**Review by jovezhong — CHANGES_REQUESTED** [review]
Thanks for the PR. The language in README is too casual. Need avoid grammar issues

File: examples/broken-stream-monitor/README.md (hunk 5 above)
Comment (gangtao) [reply examples/broken-stream-monitor/README.md:10]: random stream is already a stream

**Review by jovezhong — APPROVED** [review]

**Comment by jovezhong** [comment]
(Jove Github Bot) added it to the current sprint.

**Comment by jovezhong** [comment]
(Jove Github Bot) moved this ticket out of the GitHub project(up to 1200 tickets for one project).

**Pull request merged**